portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase

```diff
@@ -603,6 +603,94 @@ definition {
 			resourcePanels = "Portal Services,Sample QA app");
 	}
 
+	@description = "This is a use case for LRQA-57134."
+	test RefreshTokenTime {
+		property app.server.types = "jboss,tcserver,tomcat,weblogic,websphere,wildfly";
+		property database.types = "db2,hypersonic,mariadb,mysql,oracle,postgresql,sqlserver,sybase";
```

Comment:
No need to specify these two properties unless we want to limit the app servers and databases that we run this test on

Reply:
Ok I remove these lines.